source/lac/slepc_solver.cc

```diff
@@ -320,6 +320,10 @@ namespace SLEPcWrappers
     const PetscErrorCode ierr = EPSSetType (eps,
                                             const_cast<char *>(EPSKRYLOVSCHUR));
     AssertThrow (ierr == 0, ExcSLEPcError(ierr));
+
+#ifdef PETSC_HAVE_MUMPS
+    PETScWrappers::set_option_value("-st_pc_factor_mat_solver_package", "mumps");
```

Comment:
Please move it to the test. Otherwise you are forcing this always regardless to what solvers are combined with spectral transformation (which u can)

Reply:
Hmm... This just changes the default to use, isn't it? You can always override this property afterwards. In particular, this allows to run the test without further changes which seems to be advantageous. What are the disadvantages?

Reply:
No, some time ago I reworked slepc spectral transformation to accept our PETSc wrappers for solvers. With this change you override this

Reply:
It’s NOT default, it says use this no matter what

Reply:
https://www.dealii.org/8.5.0/doxygen/deal.II/classSLEPcWrappers_1_1TransformationBase.html#af238a03c00c7dbaeb60bdc6e6e710443

Reply:
So you are saying that `set_transformation` is not overriding the previous options?

Reply:
> So you are saying that set_transformation is not overriding the previous options?

you are not setting spectral transformation here, you are setting a solver to be used in spectral transformation. 

And yes, any command-like parameter will override whatever accomodations we are doing with object-oriented-proramming and making spectral transformation know about the solver via ` SLEPcWrappers::TransformationBase::::set_solver()` which internally calls `STSetKSP`.

Reply:
FYI [step-36_parallel.cc](https://github.com/dealii/dealii/blob/master/tests/slepc/step-36_parallel.cc#L287) is the only test which uses this feature. But the trick is -- you won't see the difference from outside if everything works as your eigenpairs should be the same.